README.md

```diff
@@ -41,13 +41,76 @@ to run remote actions in a recent Ubuntu container.
 
 ## Configuration
 
+### Data and log storage
+
 EngFlow stores its caches and logs in the location pointed to by the `$DATA_DIR`
 environmental variable. Mount this location onto the host to preserve data
 across container invocations and view the service logs. If remote execution is
 used, `$DATA_DIR` must have the same path on the host and in the
 container. Additionally, the Docker Unix socket must be mounted into the
 container to allow EngFlow to execute actions in Docker containers.
 
+### Setting up .bazelrc and using a custom Docker image
```

Comment:
Wouldn't it be better to recommend the "new rbe autoconfig" which is a script that you run once and then check-in the output? (which is our trick how to avoid the extra minute in the loading phase)

Reply:
Yes, however I don't know if I'm comfortable checking in that script given it uses undocumented behavior.

Reply:
Is this different from `rbe_config_gen` https://github.com/bazelbuild/bazel-toolchains#rbe_configs_gen---cli-tool-to-generate-configs? I think that's what we should be reccomending. Upstream has even deleted `rbe_autoconfig`.

Reply:
Ah, I wasn't aware of this. Yes, we should recommend this tool instead. I'll rework these instructions when I get the chance.